lib/finitomata.ex

```diff
@@ -260,6 +274,9 @@ defmodule Finitomata do
   @type transition_resolution ::
```

Comment:
Based on what I saw I would add nil to the typespec explictly (for documentation purposes). And add a information that the nil is returned upon missing callback. Unless that is going to change shortly (which I recall you talking about in the meeting)

Reply:
Nope. `t:transition_resolution/0` cannot be `nil` (it’S what gets returned from `on_transition/4` callback and it must be a tuple, otherwise you’ll get a weird error message here: https://github.com/am-kantox/finitomata/blob/huge/flow/lib/finitomata.ex#L1394